### ISSUE TITLE
`return-alt1` keybind does not work

### ISSUE THREAD

**Issue by sensasian** [jesseduffield/lazygit#2434]
**Describe the bug**
`return-alt1` keybinding config does not work

**To Reproduce**
configure as below:
```yml
# ...
keybinding:
    universal:
        return: 'q'
        return-alt1: '<c-q>'
#...
```

**Expected behavior**
Pressing `<C-q>` should exit input box such as commit prompt but does nothing.

**Screenshots**

**Version info:**
_Run `lazygit --version` and paste the result here_
`commit=16802a048e0425182f08cf0e9a2bc31480e9a55d, build date=2023-02-01T11:02:43Z, build source=binaryRelease, version=0.37.0, os=darwin, arch=arm64, git version=2.39.1`
lazygit installed via homebrew
_Run `git --version` and paste the result here_
`git version 2.39.1`

**Additional context**
I assume [this PR](https://github.com/jesseduffield/lazygit/pull/2311#issue-1489644615) introduced the `return-alt1` config. The author used similar keybindings but the other way around. Setting `return: '<c-q>'` and `return-alt1: 'q'` does work as intended, but my config does not work

**Note:** please try updating to the latest version or [manually building](https://github.com/jesseduffield/lazygit/#manual) the latest `master` to see if the issue still occurs.

**Comment by iovis** [comment]
I'm guessing this feature got removed in v0.38? I like using `q` as `return`, but for dismissing the commit prompt if I wanted to cancel it I needed to press `<esc>`.

Is there any workaround available?

**Comment by mark2185** [comment]
> I'm guessing this feature got removed in v0.38?

[Correct](https://github.com/jesseduffield/lazygit/pull/2495). 

> I like using `q` as `return`, but for dismissing the commit prompt if I wanted to cancel it I needed to press `<esc>`.
> 
> Is there any workaround available?

I think you should now be able to use `return` on all the windows/panes, except the `commit prompt` and such since that has to pass through `q` as an input to the prompt, otherwise you wouldn't be able to type `q`.

**Comment by iovis** [comment]
> I think you should now be able to use return on all the windows/panes, except the commit prompt and such since that has to pass through q as an input to the prompt, otherwise you wouldn't be able to type q.

Hahaha, yeah, that's why the `return-alt1` was useful for my use case (I realized that I cancel a lot more than I thought to double check what I'm committing haha). Thank you for clarifying!

**Comment by iovis** [comment]
> Personally I work around it by setting the git config commit.verbose to true, and then using C instead of c most of the time. That way I can at least see the diff right there in the editor, and as an added bonus I can auto-complete identifiers that occur in the diff, which is often helpful.

Yeah, I use `C` most of the time too (although I didn't know about `commit.verbose`, I should probably enable that), but sometimes I feel like it's going to be a simple commit message and then I end up wanting the context or something like that and get myself in a bind for not being able to cancel haha.

Maybe I should just rebind `C` to `c`, I think the config you mentioned is likely enough and I have an easier way to cancel from my editor anyway. Thanks for the idea!
